solidity/contracts/deposit/DepositLiquidation.sol

```diff
@@ -61,42 +61,37 @@ library DepositLiquidation {
         return (_bondValue.mul(100).div(_lotValue));
     }
 
-    /// @notice         Starts signer liquidation due to fraud.
-    /// @dev            Liquidation is done by auction.
-    /// @param  _d      Deposit storage pointer.
-    function startSignerFraudLiquidation(DepositUtils.Deposit storage _d) internal {
-        _d.logStartedLiquidation(true);
+    /// @notice           Starts signer liquidation due to abort, undercollateralization, or fraud
+    /// @dev              Liquidation is done by auction.
+    /// @param _wasFraud  Boolean checking liquidation cause. True if fraud, false otherwise.
```

Comment:
I think this could be more clearly stated as “True if liquidation is being started due to fraud, false if for any other reason.”